src/components/input/index.tsx

```diff
@@ -119,9 +124,14 @@ export class SmoothlyInput {
 		const after = this.formatter.format(StateEditor.copy(this.formatter.unformat(StateEditor.copy({ ...this.state }))))
 		this.updateBackend(after, this.inputElement)
 	}
-	// eslint-disable-next-line @typescript-eslint/no-empty-function
-	onBlur(event: FocusEvent) {}
+	onBlur(event: FocusEvent) {
+		this.smoothlyBlur.emit()
+		if (this.initialValue != this.value)
+			this.smoothlyDone.emit({ name: this.name, value: this.value })
```

Comment:
why not directly using `smoothlychanged.emit` here?

Reply:
for me it looks like repeated code

Reply:
Acctually, I changed my mind now that I saw your comment about mimicking, `input` and `change` events.
`input` vs `change` event: https://stackoverflow.com/questions/17047497/difference-between-change-and-input-event-for-an-input-element

But instead of renaming the events from how it works in HTML. Better to keep to the standard.
change names:
`smoothlyChanged` -> `smoothlyInput`
`smoothlyDone` -> `smoothlyChange`

Make sure that any component that uses `smoothlyChanged` switches to `smoothlyInput`

Reply:
⚠️ The version number in package.json should be set to `0.2.0` to signify that it is a breaking change as well. ⚠️

Reply:
Came up with a new name to keep backwards compatibility but I agree this is the better way. 
I removed all references to the `smoothlyChanged` event and changed them to use `smoothlyInput` instead as well as change `smoothlyDone` to `smoothlyChange`.